advanced/Scripts/api.sh

```diff
@@ -21,20 +21,60 @@
 TestAPIAvailability() {
 
     # as we are running locally, we can get the port value from FTL directly
-    PORT="$(pihole-FTL --config webserver.port)"
-    PORT="${PORT%%,*}"
+    local chaos_api_list availabilityResonse
 
-    availabilityResonse=$(curl -s -o /dev/null -w "%{http_code}" "http://localhost:${PORT}/api/auth")
+    # Query the API URLs from FTL using CHAOS TXT local.api.ftl
+    # The result is a space-separated enumeration of full URLs
+    # e.g., "http://localhost:80/api" "https://localhost:443/api"
+    chaos_api_list="$(dig +short chaos txt local.api.ftl @127.0.0.1)"
```

Comment:
```suggestion
    chaos_api_list="$(dig +short chaos txt local.api.ftl @127.0.0.1 > /dev/null 2>&1)"
```

Otherwise we get some unwanted output
```
dig +short chaos txt local.api.ftl @127.0.0.1
;; communications error to 127.0.0.1#53: connection refused
;; communications error to 127.0.0.1#53: connection refused
;; communications error to 127.0.0.1#53: connection refused
;; no servers could be reached
```